django/cantusdb_project/main_app/urls.py

```diff
@@ -376,6 +377,11 @@
         SourceAddImageLinksView.as_view(),
         name="source-add-image-links",
     ),
+    path(
+        "proofread/",
```

Comment:
Could this be proofread-overview or some such...since we aren't actually doing the proofreading on this view.